awx/ui_next/src/screens/Credential/CredentialDetail/CredentialDetail.jsx

```diff
@@ -1,25 +1,36 @@
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { Fragment, useState, useEffect, useCallback } from 'react';
 import { Link, useHistory } from 'react-router-dom';
 import { withI18n } from '@lingui/react';
-import { t } from '@lingui/macro';
+import { t, Trans } from '@lingui/macro';
 import { shape } from 'prop-types';
-
+import styled from 'styled-components';
 import { Button, List, ListItem } from '@patternfly/react-core';
 import AlertModal from '../../../components/AlertModal';
 import { CardBody, CardActionsRow } from '../../../components/Card';
 import ContentError from '../../../components/ContentError';
 import ContentLoading from '../../../components/ContentLoading';
 import DeleteButton from '../../../components/DeleteButton';
 import {
-  DetailList,
   Detail,
+  DetailList,
   UserDateDetail,
 } from '../../../components/DetailList';
+import ChipGroup from '../../../components/ChipGroup';
+import CodeMirrorInput from '../../../components/CodeMirrorInput';
+import CredentialChip from '../../../components/CredentialChip';
 import ErrorDetail from '../../../components/ErrorDetail';
 import { CredentialsAPI, CredentialTypesAPI } from '../../../api';
 import { Credential } from '../../../types';
 import useRequest, { useDismissableError } from '../../../util/useRequest';
 
+const PluginInputMetadata = styled(CodeMirrorInput)`
```

Comment:
aren't codemirrors already full-length?

Reply:
If we don't make it take up all the columns of the grid then it ends up looking like:

<img width="1366" alt="Screen Shot 2020-06-09 at 4 55 13 PM" src="https://user-images.githubusercontent.com/9889020/84198782-0f6cbe80-aa72-11ea-85c6-877f5ad32049.png">